cylc/uiserver/handlers.py

```diff
@@ -28,7 +30,55 @@
 from .websockets import authenticated as websockets_authenticated
 
 
-class BaseHandler(web.RequestHandler):
+logger = logging.getLogger(__name__)
+
+
+ME = getpass.getuser()
+
+
+def _authorised(req):
+    """Authorise a request.
+
+    Requires the request to be authenticated.
+
+    Currently returns True if the authenticated user is the same as the
+    user under which this UI Server is running.
+
+    Returns:
+        bool - True if the request passes authorisation, False if it fails.
+
+    """
+    user = req.get_current_user()
+    username = user.get('name', '?')
+    if username != ME:
+        logger.warn(f'Authorisation failed for {username}')
```

Comment:
`.warn` is deprecated in logging.

```suggest
       logger.warning(f'Authorisation failed for {username}')
```

Reply:
https://docs.python.org/3/library/logging.html#logging.warning

Reply:
(Checked we haven't used this anywhere else :+1: )

Reply:
And deprecated in 3.3: https://bugs.python.org/issue13235